src/main_thread/init/utils/update_manifest_codec_support.ts

```diff
@@ -131,6 +131,11 @@ export function updateManifestCodecSupport(
       let hasSupportedCodec: boolean = false;
       let hasCodecWithUndefinedSupport: boolean = false;
       adaptation.representations.forEach((representation) => {
+        if (representation.isCodecSupportedInWebWorker === false) {
+          representation.isSupported = false;
```

Comment:
A relatively minor thing is bothering me here.

Technically, there's nothing in the semantics of `isCodecSupportedInWebWorker` that indicates that playback will actually currently-go through MSE-in-worker.
We could even in the future have e.g. a fallback to main thread or to multi-thread without MSE-in-worker while that property is set at its current meaning.

Shouldn't we be checking both that property and whether we're playing with MSE-in-worker explicitely?